core/src/main/java/org/jruby/util/Pack.java

```diff
@@ -542,7 +561,7 @@ private static ByteList encodes(Ruby runtime, ByteList io2Append,byte[]charsToEn
      * @param iLength The max number of characters to encode
      * @return the io2Append buffer
      **/
-    private static ByteList qpencode(ByteList io2Append, ByteList i2Encode, int iLength) {
+    public static ByteList qpencode(ByteList io2Append, ByteList i2Encode, int iLength) {
```

Comment:
It might be better to move this out to a `PackSupport` or `PackUtils` like we've done with `RubyString`.  While that was mostly to divorce from `org.jruby.RubyString` it was also done to prevent people thinking it was now a public part of the API.